roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,157 @@
+"""
+Module for performing checks on a Fluentd logging deployment configuration
+"""
+
+import json
+
+from openshift_checks import OpenShiftCheck, get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+
+
+class FluentdConfig(OpenShiftCheck):
```

Comment:
Not sure if we also want to limit this check to only run on the first master. If we don't, we would have to override the parent's `is_active` method anyway

cc @sosiouxme